components/vf-grid-page/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 1.0.1
```

Comment:
I think we should make this a 2.0.0 release as it will require sprinkling vf-u-fullbleed around or require updating other components, and perhaps might cause other layout breaks.

For example, `vf-divider` is no longer full-width -- or is there a separate PR for what to do with that?

Reply:
it would also require the removal of `vf-u-grid-reset` as it should no longer be needed

Reply:
I think it comes down to some good documentation -- "you need to make sure vf-body is on the body element" -- as best practice.